rpc/core/tx.go

```diff
@@ -18,10 +18,10 @@ import (
 )
 
 const (
-	txStatusUnknown   string = "UNKNOWN"
-	txStatusPending   string = "PENDING"
-	txStatusEvicted   string = "EVICTED"
-	txStatusCommitted string = "COMMITTED"
+	TxStatusUnknown   string = "UNKNOWN"
+	TxStatusPending   string = "PENDING"
+	TxStatusEvicted   string = "EVICTED"
+	TxStatusCommitted string = "COMMITTED"
```

Comment:
code in this PR needs to be updated to use the renamed constants

Reply:
ups i did it on the go and didn't have a chance to double check! thanks rootul! should be fixed now! :)